### PR TITLE
Add doc to represent work done for pending GEPs as of November 2023

### PR DESCRIPTION
**Pull request by candita**
**What type of PR is this?**

/kind documentation
/area conformance

**What this PR does / why we need it**:
Create a point-in-time record for the status of provisional and experimental GEPs as of November 2023.

**Which issue(s) this PR fixes**:

None, but I can add one.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by sjberman** [comment]
Could NGINX Gateway Fabric be added to this table? Right now the only GEP that we support is `GEP-1709: Conformance Profiles`.

File: site-src/geps/pending-geps.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| _Compliance_        | NS                            | NS                                   | NS                                 |                     |                                       |                                |                               |                                         |                                |                              |                                                               | NS                         |                                      |                                                     |                                       |
+| **EKS**             |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
```
Comment (robscott) [inline site-src/geps/pending-geps.md:40]: Nit: I'd recommend sorting these alphabetically by the name as shown in this table.

File: site-src/geps/pending-geps.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* PP - Planned, in progress
+* RR - Ready in next release
+* C  - Complete in current release
+
+For example, an imaginary implementor called Imaginary has support completed for GRPC Route (GEP-1016), planned
+ and in progress for HTTPRoute Timeouts (GEP-1742), but otherwise not planned for any other GEP.
+Imaginary's row in the table would look like this:
+
+
+|      | GEP-1619: Session Persistence | GEP-1867: Per-Gateway Infrastructure | GEP-957: Destination Port Matching | GEP-1016: GRPCRoute | GEP-1324: Service Mesh in Gateway API | GEP-1426: xRoutes Mesh Binding | GEP-1651: Gateway Routability | GEP-1686: Mesh conformance testing plan | GEP-1709: Conformance Profiles | GEP-1742: HTTPRoute Timeouts | GEP-1748: Gateway API Interaction with Multi-Cluster Services | GEP-1897: BackendTLSPolicy | GEP-1911: Backend Protocol Selection | GEP-2162: Supported features in GatewayClass Status | GEP-2257: Gateway API Duration Format |
```
Comment (robscott) [inline site-src/geps/pending-geps.md:27]: I don't think it's worth including provisional or unimplementable GEPs in the list (That would exclude persistence, duration format, conformance profiles, and routability).

Many of the remaining GEPs have corresponding supported features, so we could get data from conformance reports (https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports) to help populate them.

File: site-src/geps/pending-geps.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#### Gateway API Enhancment Proposal Status November 2023
```
Comment (robscott) [inline site-src/geps/pending-geps.md:1]: I think this is a useful idea, and I'm really not sure how else to track this, but I'm hesitant to add it in this location (our official docs). Maybe it belongs in `hack/` for lack of a better place? Just something to show that this is a temporary/working doc.

File: site-src/geps/pending-geps.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+| **Gloo**            |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
+| **GKE**             |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
```
Comment (robscott) [inline site-src/geps/pending-geps.md:50]: GKE currently supports "GEP-1748: Gateway API Interaction with Multi-Cluster Services", the rest are all "not planned" at this point as we only support the standard channel.

**Review by robscott — COMMENTED** [review]
Thanks for setting this up @candita!

**Comment by candita** [comment]
> Could NGINX Gateway Fabric be added to this table? Right now the only GEP that we support is `GEP-1709: Conformance Profiles`.

@sjberman Yes, please create suggestion comment with your changes, and I will accept/resolve it to add.

File: site-src/geps/pending-geps.md (hunk 1 above)
Comment (candita) [reply site-src/geps/pending-geps.md:40]: They were alphabetical by real name, but I had shortened them here.  I will add the real name.

File: site-src/geps/pending-geps.md (hunk 4 above)
Comment (candita) [inline site-src/geps/pending-geps.md:50]:
```suggestion
| **GKE**             |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |      C                                                         |                            |                                      |                                                     |                                       |
```

Comment (candita) [reply site-src/geps/pending-geps.md:50]: I'll make the change.

File: site-src/geps/pending-geps.md (hunk 3 above)
Comment (candita) [reply site-src/geps/pending-geps.md:1]: I'll move to hack/

File: site-src/geps/pending-geps.md (hunk 2 above)
Comment (candita) [reply site-src/geps/pending-geps.md:27]: I imagined using this to gauge interest and planning for the provisional/experimental GEPs.  Only the first two are marked as provisional in the GEPS.  I think only the conformance profiles is unimplementable, but still it warrants interest and participation.   Which others are unimplementable?

Also, i was hoping to get group participation here, only saw a handfu (5) conformance reports.

File: site-src/geps/pending-geps.md (hunk 4 above)
Comment (candita) [reply site-src/geps/pending-geps.md]: But GEP-1748 is still listed as experimental.

**Comment by candita** [comment]
@robscott @shaneutt @youngnick I will close this due to lack of interest.  We can reopen if warranted.

**Pull request closed without merging**